ballerina/utils.bal

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// This class is used by the native implementation to add the MQRFH2 fields to a table.
+class NativeUtils {
```

Comment:
Had to invoke a bal method from native side due to -https://github.com/ballerina-platform/ballerina-lang/issues/41660

Reply:
Shall we mark this class `isolated` ?